components/org.wso2.carbon.identity.user.endpoint/src/main/resources/api.identity.user.yaml

```diff
@@ -58,30 +58,31 @@ paths:
   /me:
     post:
       description: |
-         This API is used to user self registration.
+         This API is used for user self registration.
       x-wso2-request: |
-        curl -X POST -H "Authorization: Basic YWRtaW46YWRtaW4=" -H "Content-Type: application/json" -d '{"user": {"username": "isura","realm": "ENGINEERING", "password": "P@ssw0rd2017","claims": [{"uri": "http://wso2.org/claims/givenname","value": "Isura" },{"uri": "http://wso2.org/claims/emailaddress","value": "isura@mail.com"},{"uri": "http://wso2.org/claims/lastname","value": "Karunaratne"},{"uri": "http://wso2.org/claims/mobile","value": "+947721584558"} ] },"properties": []}' "https://localhost:9443/api/identity/user/v1.0/me"
-
+        curl -X POST -H "Authorization: Basic <Base64Encoded_access_token>" -H "Content-Type: application/json" -d '{"user": {"username": "<username>","realm": "<user_store>", "password": "<password>","claims": [{"uri": "<claim_URI>","value": "<claim_value>" },{"uri": "<claim_URI2>","value": "<claim_value2>"},{"uri": "<claim_URI3>","value": "<claim_value3>"},{"uri": "<claim_URI4>","value": "<claim_value4>"} ] },"properties": []}' "https://localhost:9443/api/identity/user/v1.0/me"
```

Comment:
This could be either a base 64 encoded access token or base64 encoded <username:password>
For evaluation purposes mostly the Http basic authentication scheme which is the base64 encoded <username:password> will be used.

Can we pitch that accordingly here.

Reply:
Fixed with commit https://github.com/wso2-extensions/identity-governance/pull/232/commits/d216a057bad7de8a055411548189bd87b11d98b5